### PR TITLE
Bug 1849189 - Update Focus telemetry probes expiring in v120

### PR DESCRIPTION
**Pull request by rocketsroger**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1849189

### CODE REVIEW

**Comment by rocketsroger** [comment]
# Request for Data Collection Renewal
### Renew for Never
Total: 31
———
`browser.is_default`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`browser.default_search_engine`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`mozilla_products.has_fenix_installed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5303

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`shortcuts.shortcuts_on_home_number`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5189

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`shortcuts.shortcut_opened_counter`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5189

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`shortcuts.shortcut_added_counter`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5189

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`shortcuts.shortcut_removed_counter`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5189

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.toolbar_shield_clicked`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.tracking_protection_changed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.tracker_setting_changed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.has_social_blocked`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.has_advertising_blocked`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.has_analytics_blocked`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`tracking_protection.has_content_blocked`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5163

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`onboarding.first_screen_close_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`onboarding.get_started_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`onboarding.default_browser_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`onboarding.skip_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`onboarding.second_screen_close_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`search_widget.add_to_home_screen_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`search_widget.widget_was_added`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.visited_setting`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/8008#issuecomment-1322266028

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.setting_changed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/8008#issuecomment-1322266028

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.cookie_banner_cfr_shown`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.exception_added`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.report_site_domain`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.report_site_cancel_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.report_domain_site_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.exception_removed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`cookie_banner.visited_panel`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866

2) When will this collection now expire?
    - Never

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———

**Comment by rocketsroger** [comment]
# Request for Data Collection Renewal
### Renew for 3 months
Total: 5
———
`browser.locale_override`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`metrics.search_widget_installed`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7474

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`search_widget.new_tab_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7474

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`search_widget.voice_button`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7474

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———
`search_widget.promote_dialog_shown`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947

2) When will this collection now expire?
    - 123

3) Why was the initial period of collection insufficient?
    - This is important to understand user behavior

———

File: focus-android/app/metrics.yaml [hunk 1]
```diff
@@ -56,11 +60,12 @@ browser:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647
       - https://github.com/mozilla-mobile/focus-android/pull/7418#issuecomment-1195518264
+      - https://github.com/mozilla-mobile/firefox-android/pull/3320
     data_sensitivity:
       - technical
     notification_emails:
       - android-probes@mozilla.com
-    expires: 119
+    expires: 123
```
Comment (t-p-white) [inline focus-android/app/metrics.yaml:68]: I don't see this probe in the Request for Data Collection Renewal - should it be?

File: focus-android/app/metrics.yaml [hunk 2]
```diff
@@ -762,11 +791,12 @@ preferences:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5526
       - https://github.com/mozilla-mobile/focus-android/pull/7418#issuecomment-1195518264
+      - https://github.com/mozilla-mobile/firefox-android/pull/3320
     data_sensitivity:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-    expires: 119
+    expires: 123
```
Comment (t-p-white) [inline focus-android/app/metrics.yaml:799]: I don't see `user_theme` in the Request for Data Collection Renewal - should it be?

**Review by t-p-white — COMMENTED** [review]
Looks good, just one comment from me around the `user_theme` probe

File: focus-android/app/metrics.yaml (hunk 2 above)
Comment (rocketsroger) [reply focus-android/app/metrics.yaml:799]: Good catch.  Missed it when creating the request.  I've updated the 3 months request to include the `user_theme` probe.  Thanks!

**Review by t-p-white — APPROVED** [review]

**Comment by rocketsroger** [comment]
> # Request for Data Collection Renewal
> ### Renew for Never
> 
> Total: 31 ——— `browser.is_default`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Issue #4901: Update metrics for data review on initial metrics focus-android#5065 (comment)](https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `browser.default_search_engine`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Issue #4901: Update metrics for data review on initial metrics focus-android#5065 (comment)](https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `mozilla_products.has_fenix_installed`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5295: Add metrics for other Mozilla products. focus-android#5303](https://github.com/mozilla-mobile/focus-android/pull/5303)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `shortcuts.shortcuts_on_home_number`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For 5056: Add Telemetry for Shortcuts. focus-android#5189](https://github.com/mozilla-mobile/focus-android/pull/5189)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `shortcuts.shortcut_opened_counter`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For 5056: Add Telemetry for Shortcuts. focus-android#5189](https://github.com/mozilla-mobile/focus-android/pull/5189)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `shortcuts.shortcut_added_counter`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For 5056: Add Telemetry for Shortcuts. focus-android#5189](https://github.com/mozilla-mobile/focus-android/pull/5189)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `shortcuts.shortcut_removed_counter`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For 5056: Add Telemetry for Shortcuts. focus-android#5189](https://github.com/mozilla-mobile/focus-android/pull/5189)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.toolbar_shield_clicked`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.tracking_protection_changed`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.tracker_setting_changed`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.has_social_blocked`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.has_advertising_blocked`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.has_analytics_blocked`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `tracking_protection.has_content_blocked`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5057: Add Telemetry for Tracking Protection. focus-android#5163](https://github.com/mozilla-mobile/focus-android/pull/5163)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `onboarding.first_screen_close_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7500 Create onBoarding screen one and two focus-android#7566 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `onboarding.get_started_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7500 Create onBoarding screen one and two focus-android#7566 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `onboarding.default_browser_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7500 Create onBoarding screen one and two focus-android#7566 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `onboarding.skip_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7500 Create onBoarding screen one and two focus-android#7566 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `onboarding.second_screen_close_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7500 Create onBoarding screen one and two focus-android#7566 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7566#issuecomment-1235551604)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `search_widget.add_to_home_screen_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7506 promote search widget dialog focus-android#7657 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `search_widget.widget_was_added`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7506 promote search widget dialog focus-android#7657 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.visited_setting`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7965 - Handle Cookie Banner in Focus focus-android#8008 (comment)](https://github.com/mozilla-mobile/focus-android/pull/8008#issuecomment-1322266028)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.setting_changed`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7965 - Handle Cookie Banner in Focus focus-android#8008 (comment)](https://github.com/mozilla-mobile/focus-android/pull/8008#issuecomment-1322266028)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.cookie_banner_cfr_shown`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * https://github.com/mozilla-mobile/focus-android/
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.exception_added`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #1797578 - Create toggle option in the toolbar that allow a site to be excluded from cookie banner handling in Focus focus-android#8124 (comment)](https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.report_site_domain`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Bug 1803589 - Implement submit site support requests for cookie banners in Focus #389 (review)](https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.report_site_cancel_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Bug 1803589 - Implement submit site support requests for cookie banners in Focus #389 (review)](https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.report_domain_site_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Bug 1803589 - Implement submit site support requests for cookie banners in Focus #389 (review)](https://github.com/mozilla-mobile/firefox-android/pull/389#pullrequestreview-1341440145)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.exception_removed`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #1797578 - Create toggle option in the toolbar that allow a site to be excluded from cookie banner handling in Focus focus-android#8124 (comment)](https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ——— `cookie_banner.visited_panel`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #1797578 - Create toggle option in the toolbar that allow a site to be excluded from cookie banner handling in Focus focus-android#8124 (comment)](https://github.com/mozilla-mobile/focus-android/pull/8124#issuecomment-1344449866)
> 
>     2. When will this collection now expire?
>        
>        * Never
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior
> 
> 
> ———

# Data Collection Renewal Review

1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

Yes

2) Is the data collection covered by the existing Firefox Privacy Notice?

Yes

## Result

data-review+

**Comment by rocketsroger** [comment]
> # Request for Data Collection Renewal
> ### Renew for 3 months
> 
> Total: 6 ——— `browser.locale_override`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Issue #4901: Update metrics for data review on initial metrics focus-android#5065 (comment)](https://github.com/mozilla-mobile/focus-android/pull/5065#issuecomment-894328647)
> 
>     2. When will this collection now expire?
>        
>        * 123
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
>          ———
>          `preferences.user_theme`:
> 
>     4. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #5519: Add telemetry for theme settings. focus-android#5526](https://github.com/mozilla-mobile/focus-android/pull/5526)
> 
>     5. When will this collection now expire?
>        
>        * 123
> 
>     6. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
>          ———
>          `metrics.search_widget_installed`:
> 
>     7. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7249 Create a search widget focus-android#7474](https://github.com/mozilla-mobile/focus-android/pull/7474)
> 
>     8. When will this collection now expire?
>        
>        * 123
> 
>     9. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
> 
> 
> ——— `search_widget.new_tab_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7249 Create a search widget focus-android#7474](https://github.com/mozilla-mobile/focus-android/pull/7474)
> 
>     2. When will this collection now expire?
>        
>        * 123
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
> 
> 
> ——— `search_widget.voice_button`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7249 Create a search widget focus-android#7474](https://github.com/mozilla-mobile/focus-android/pull/7474)
> 
>     2. When will this collection now expire?
>        
>        * 123
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
> 
> 
> ——— `search_widget.promote_dialog_shown`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [For #7506 promote search widget dialog focus-android#7657 (comment)](https://github.com/mozilla-mobile/focus-android/pull/7657#issuecomment-1252242947)
> 
>     2. When will this collection now expire?
>        
>        * 123
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * This is important to understand user behavior, will remove in 3 more releases.
> 
> 
> ———

# Data Collection Renewal Review

1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

Yes

2) Is the data collection covered by the existing Firefox Privacy Notice?

Yes

## Result

data-review+

**Pull request merged**